benchmarks-next/README.md

```diff
@@ -0,0 +1,106 @@
+# Candidate Benchmark Programs
+
+This directory contains the candidate programs for the benchmark suite. They are
+candidates, not officially part of the suite yet, because we [intend][rfc] to
+record various metrics about the programs and then run a principal component
+analysis to find a representative subset of candidates that doesn't contain
+effectively duplicate workloads.
+
+[rfc]: https://github.com/bytecodealliance/rfcs/pull/4
+
+## Building
+
+Build an individual benchmark program via:
+
+```
+$ cargo run -p sightglass-cli -- build path/to/benchmark/Dockerfile
+```
+
+Build all benchmark programs by entering this directory
+(`sightglass/benchmarks-next`) and running:
+
+```
+$ ./build.sh
+```
+
+## Minimal Technical Requirements
+
+In order for the benchmark runner to successfully execute a Wasm program and
+record its execution, it must:
+
+* Export a `_start` function of type `[] -> []`.
+
+* Import `bench.start` and `bench.end` functions, both of type `[] -> []`.
+
+* Call `bench.start` exactly once during the execution of its `_start`
+  function. This is when the benchmark runner will start recording execution
+  time and performance counters.
+
+* Call `bench.end` exactly once during execution of its `_start` function, after
+  `bench.start` has already been called. This is when the benchmark runner will
+  stop recording execution time and performance counters.
+
+* Not use `stdout`. It may use `stderr` for logging, if necessary.
```

Comment:
Why, though? If the reason is for us to be able to distinguish between sightglass output and benchmark output, we should explain that here.